.dir-locals.el

```diff
@@ -1,5 +1,9 @@
 (
  (haskell-mode . ((dante-target . "tiny-lang:lib")))
- ("test" . ((haskell-mode . ((dante-target . "tiny-lang:tiny-lang-test")))))
+ ("test" . ((haskell-mode . ((dante-target . "tiny-lang:tiny-lang-test")
+			     (haskell-process-args-stack-ghci . ("--ghci-options=-ferror-spans"
+								  "--no-build"
+								  "--no-load"
+								  "--test"))))))
```

Comment:
BTW, I'm curious, what does this do?

Reply:
It sets up the necessary flags for `interactive-haskell-mode`.

I usually work in `dante`, but sometimes I like to load everything to ghci and experiment with it.

This ensures that when you do `C-c C-l` or `(haskell-process-load-file)`, then it will also load the packages specified for testing.